src/nav/apm.yml

```diff
@@ -1349,14 +1349,10 @@ pages:
             path: /docs/apm/apm-ui-pages/events/thread-profiler-tool
       - title: Error analytics
         pages:
-          - title: Errors page
+          - title: Errors 
```

Comment:
so am I understanding correctly we have the doc in two places in the navigation. that's a pattern we try to avoid as much as possible, if you can figure out a way to avoid that and have it in a single place.

Reply:
Yes... I agree. This is something I need to sort out on a wider nav level that I haven't totally figured out how to handle.